yb-voyager/cmd/importData.go

```diff
@@ -595,7 +595,7 @@ func importData(importFileTasks []*ImportFileTask) {
 				maxColocatedBatchesInProgress := utils.GetEnvAsInt("YBVOYAGER_MAX_COLOCATED_BATCHES_IN_PROGRESS", 3)
 				err := importTasksViaTaskPicker(pendingTasks, state, progressReporter, maxParallelConns, maxParallelConns, maxColocatedBatchesInProgress)
 				if err != nil {
-					utils.ErrExit("Failed to import tasks via task picker: %s", err)
+					utils.ErrExit("Failed to import tasks via task picker; %s", err)
```

Comment:
this is so that we can get more error context in callhome. I verified that all errors returned by the function `importTasksViaTaskPicker` do not contain any sensitive information.

Reply:
what exactly do we do before sending to callhome? do we break on colon, can you point me to that code.

Reply:
`SanitizeErrorMsg`

Reply:
Got it.
Semi colon seems a bit off, I was thinking if we can use hypen `-`
or there are some variations of colon also can be used without affecting the output (in case of ErrExit)

https://go.dev/play/p/KKm2-SP1TBm

Reply:
I think hypen or even full-stop is better. let me change it